app/assets/javascripts/admin/addon/models/web-hook.js

```diff
@@ -41,7 +41,7 @@ export default class WebHook extends RestModel {
   verify_certificate = true;
   active = false;
   web_hook_event_types = null;
-  groupsFilterInName = null;
+  group_names = null;
```

Comment:
Just took the chance to rename this. A global search in public and private repos shows this isn't used anywhere else.